components/social/SocialCard.js

```diff
@@ -20,7 +20,7 @@ const SocialCard = ({ ogImageUrl, username, loading }) => {
 
   return (
     <motion.div
-      className="flex flex-col justify-between gap-4 py-3 px-3 md:px-5 shadow-bs1 rounded-md h-full w-full min-h-[442px] bg-white"
+      className="flex flex-col justify-between gap-4 py-3 px-3 md:px-5 shadow-bs1 rounded-md h-full w-full min-h-[350px] bg-white"
```

Comment:
Hi @josecollado needed a few changes

- [ ] `min-h-[350px]` will be over `md:` only. So it will be `md:min-h-[350px]`
- [ ] In `/components/social/SocialShare.js ` line 36 `mb-4` will be over `md:` only. So it will be `md:mb-4` 

I think these will help to enhance the card more for mobile screen